### DISCUSSION TITLE
Q: Rendering emojis from OS.

### DISCUSSION THREAD

**Discussion by nathan130200** [SixLabors/ImageSharp#2038] [Show and tell] [answered]
Hello, i have an question about Imagesharp. Is able to render emojis from fonts?

**Comment by JimBobSquarePants** [comment, marked as answer]
Yes. See https://github.com/SixLabors/Fonts

**Reply by JimBobSquarePants** [reply]
Used via https://github.com/SixLabors/ImageSharp.Drawing

**Reply by nathan130200** [reply]
Omg thanks!
